python/doc/theory/numerical_methods/rosenblatt_transformation.rst

```diff
@@ -92,17 +95,14 @@ where both transformations :math:`T_1`, and :math:`T_2` are given by:
          \right)
        \end{array}
 
-| where :math:`F_{k|1, \dots, k-1}` is the cumulative distribution
-  function of the conditional random variable
-  :math:`X_k|X_1, \dots, X_{k-1}` and :math:`\Phi` the cumulative
-  distribution function of the standard :math:`1`-dimensional Normal
-  distribution.
+where :math:`F_{k|1, \dots, k-1}` is the cumulative distribution function of the conditional
+random variable :math:`X_k|X_1, \dots, X_{k-1}` and :math:`\Phi` is the cumulative distribution
+function of the standard :math:`1`-dimensional Normal distribution.
 
 
 .. topic:: API:
 
-    - See the available :ref:`Rosenblatt transformations <isoprobabilistic_transformations>`.
-
+    - See  :ref:`Rosenblatt transformation <isoprobabilistic_transformations>`.
 
 
 .. topic:: References:
```

Comment:
The references are poorly formatted:

![image](https://user-images.githubusercontent.com/31351465/235193705-c1c45c85-d237-4553-9307-66c7be3c0460.png)

This is because the references are written on a single line that RST cannot format.

If we split the line and write the remaining lines of each item aligned with the first:

```
.. topic:: References:

    - O. Ditlevsen and H.O. Madsen, 2004, "Structural reliability methods," 
      Department of mechanical engineering technical university of Denmark - 
      Maritime engineering, internet publication.
    - J. Goyet, 1998, "Sécurité probabiliste des structures - 
      Fiabilité d'un élément de structure," Collège de Polytechnique.
    - A. Der Kiureghian, P.L. Liu, 1986,"Structural Reliability Under 
      Incomplete Probabilistic Information", Journal of Engineering Mechanics, 
      vol 112, no. 1, pp85-104.
    - [lebrun2009a]_
    - [lebrun2009b]_
    - [lebrun2009c]_
    - H.O. Madsen, Krenk, S., Lind, N. C., 1986, "Methods of Structural 
      Safety," Prentice Hall.
    - [nataf1962]_

```
it produces:
![image](https://user-images.githubusercontent.com/31351465/235195475-0dabd7f0-d417-4c56-9e1e-f80512b46bd4.png)